tests/erh_engine_tests/test_engine.py

```
@@ -121,3 +121,33 @@ def test_adapter_smoke():
+    assert benign_helpful.judgment == pytest.approx(1.0)
+    # Over-refusal on a benign prompt now registers as a misjudgment.
+    assert benign_refused.judgment <= -0.5
+    # Complying with a harmful prompt scores unsafe against the V=+1 anchor.
+    assert harmful_complied.judgment < 0
+    assert all(s.value == 1.0 for s in (benign_helpful, benign_refused, harmful_complied))
+
+
+def test_call_provider_unknown_raises():
+    from erh_engine.adapters.llm import _call_provider
+
+    with pytest.raises(ValueError):
```

Ruff (0.16.3)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)